sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/ImportJobDigitalTwinOptions.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.Context;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The optional parameters for
+ * {@link com.azure.digitaltwins.core.DigitalTwinsClient#listImportJobs(ImportJobDigitalTwinOptions, Context)} and
+ * {@link com.azure.digitaltwins.core.DigitalTwinsAsyncClient#listImportJobs(ImportJobDigitalTwinOptions)}
+ */
+@Fluent
+public final class ImportJobDigitalTwinOptions {
```

Comment:
I would suggest renaming this to `DigitalTwinsImportJobOptions` to mimic naming patterns used in other contexts.

Reply:
Fixed it.